devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy

```diff
@@ -23,6 +23,7 @@ metadata {
 		capability "Temperature Alarm"
 
 		fingerprint mfr:"010F", prod:"0C02", model:"1002"
+		fingerprint mfr:"010F", prod:"0C02", model:"3002", deviceId: "0x0701", inClusters:"0x5E,0x86,0x72,0x59,0x73,0x80,0x22,0x31,0x56,0x98,0x7A"
```

Comment:
just include the mfr, prod, and model fields.

Those should be enough to uniquely specify the device

Reply:
Ok, changes done. Please check.